js/hubs/components/ReleasePurchase.js

```diff
@@ -50,7 +49,6 @@ const ReleasePurchase = (props) => {
   const [release, setRelease] = useState(undefined)
   const [userIsRecipient, setUserIsRecipient] = useState(false)
   const [publishedHub, setPublishedHub] = useState()
```

Comment:
I think we can kill all this state logic and the 'published via..' copy since publishedHub is never set anywhere

Reply:
not in scope for this pr